scripts/generator/graphql_generator/csharp/SDK/WebCheckoutMessageReceiver.cs.erb

```diff
@@ -64,6 +64,7 @@ namespace <%= namespace %>.SDK {
                     var checkout = (Checkout) response.node();
                     if (checkout.completedAt() != null) {
                         OnSuccess();
+                        Destroy(this);
```

Comment:
This will only destroy the script and not the associated game object, is this intended?